rest-api-spec/src/main/resources/rest-api-spec/api/streams.status.json

```diff
@@ -0,0 +1,35 @@
+{
+  "streams.status": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/streams-status.html",
+      "description": "Return the current status of the streams feature for each streams type"
+    },
+    "stability": "stable",
+    "visibility": "public",
+    "headers": {
+      "accept": [
+        "application/json"
+      ]
+    },
+    "url": {
+      "paths": [
+        {
+          "path": "/_streams/status",
+          "methods": [
+            "GET"
+          ]
+        }
+      ]
+    },
+    "params": {
+      "local": {
+        "type": "boolean",
+        "description": "Return local information, do not retrieve the state from master node (default: false)"
+      },
+      "master_timeout": {
+        "type": "time",
+        "description": "Specify timeout for connection to master"
+      }
```

Comment:
It doesn't look like we actually support either of these (I get `request [/_streams/status] contains unrecognized parameter` when I try either).

Reply:
Hummmm I thought these timeouts came for free when extending AcknowladgedRequest. Let me have a dig and see what I missed.

Reply:
OK I thought these were "Common Query Params" but it seems they are not. I have updated the REST actions to explicitly declare them as supported params and updated the YAML spec to be more accurate.